proof/refine/AARCH64/Syscall_R.thy

```diff
@@ -2086,7 +2086,7 @@ crunches handleVMFault
 
 crunches handleHypervisorFault
   for ksit[wp]: "\<lambda>s. P (ksIdleThread s)"
-  (wp: undefined_valid simp: isFpuEnable_def)
+  (wp: undefined_valid haskell_assert_inv simp: isFpuEnable_def)
```

Comment:
didn't you already put `haskell_assert_inv` into `[wp]`?

Reply:
`haskell_assert_wp` is in the `[wp]` set

Reply:
Right! my bad.